src/langsmith/deployments-scale.mdx

```diff
@@ -0,0 +1,353 @@
+| <Tooltip tip="Number of read requests being processed by the Deployment per second">Read requests per second</Tooltip> | 5 | 500 | 5 | 50 | 500 |
+| **API replicas**<br />(1 CPU, 2Gi per replica) | 1 (default) | 6 | 10 | 3 | 15 |
+| **Queue replicas**<br />(1 CPU, 2Gi per replica) | 1 (default) | 10 | 1 (default) | 5 | 10 |
+| **N_JOBS_PER_WORKER** | 10 (default) | 50 | 10 | 10 | 50 |
+| **Redis resources** | 2 Gi (default) | 2 Gi (default) | 2 Gi (default) | 2 Gi (default) | 2 Gi (default) |
+| **Postgres resources** | 2 CPU<br />8 Gi (default) | 4 CPU<br />16 Gi memory | 4 CPU<br />16 Gi per replica | 4 CPU<br />16 Gi memory | 8 CPU<br />32 Gi memory |
+
+Below we go into more details about the request handling and scaling patterns as well as provide configuration examples for your LangSmith Deployment.
+
+## Scaling for write load
+
+Write load is primarily driven by the following factors:
+
+- Creation of new runs
+- Creation of new checkpoints during run execution
+- Writing to long term memory
+- Creation of new threads
+- Creation of new assistants
+- Execution of periodic cron jobs
+
+The following systems are primarily responsible for handling write load:
+
+- API server: Handles initial request and persistence of data to the database.
+- Queue: Handles the execution of runs.
+- Redis: Handles the storage of ephemeral data about on-going runs.
+- Postgres: Handles the storage of all data, including run, thread, assistant, cron job, checkpointing and long term memory
+
+### Best practices for scaling the write path
+
+#### Enable the use of queue replicas
+
+By default, the api replica manages the queue and does not use queue replicas. You can enable the use of queue replicas by setting the `queue.enabled` configuration to `true`.
+
+```yaml
+queue:
+  enabled: true
+```
+
+This will allow the api replica to offload the queue management to the queue replicas, significantly reducing the load on the api replica and allowing it to focus on handling requests.
+
+#### Change N_JOBS_PER_WORKER based on assistant characteristics
+
+The default value of N_JOBS_PER_WORKER is 10. You can change this value to scale the maximum number of runs that can be executed at a time by a single queue replica based on the characteristics of your assistant.
+
+Some general guidelines for changing N_JOBS_PER_WORKER:
+
+- If your assistant is computationally intensive, lower N_JOBS_PER_WORKER to handle fewer concurrent runs per replica or keep it at the default.
+- If your assistant is computationally lightweight, increase N_JOBS_PER_WORKER to handle more concurrent runs per replica.
+
+There is no upper limit to N_JOBS_PER_WORKER. However, the polling algorithm queues use to pick up new runs is greedy, meaning they will try to pick up as many runs as possible and begin executing them right away. If N_JOBS_PER_WORKER is too high and traffic is burstly, you may see uneven replica utilization and potentially longer run execution times.
```

Comment:
Nit: `queues use` --> this reads slightly weird. How about just `the polling algorithm to pick up new runs is greedy`?